google/cloud/forseti/services/inventory/storage_helpers.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for data transformation."""
+
+from google.cloud.forseti.services.inventory.storage import DataAccess
+from google.cloud.forseti.services.inventory.storage import Storage
+
+
+def get_user_emails(service_config, member_types=None):
```

Comment:
It's not clear that this needs to be in a separate module. Either it's generally useful, and it should be written in a way that it is generic and solves this problem for multiple uses, or it is specific to your scanner and should just be a private function defined within the context of the scanner.